<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SG-LT-122" ignored="1" name="Default test(failed)" tests="2" failures="1" timestamp="2020-10-17T19:49:14 IST" time="0.101" errors="0">
  <testcase name="test2" time="0.015" classname="demo.TestNGRetryFailedDemo"/>
  <testcase name="test3" time="0.008" classname="demo.TestNGRetryFailedDemo">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at demo.TestNGRetryFailedDemo.test3(TestNGRetryFailedDemo.java:23)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase name="test1" time="0.0" classname="demo.TestNGRetryFailedDemo">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
